File: src/main/java/service/CandidatoService.java
package service;

import dao.CandidatoDao;
import model.Candidato;

import javax.inject.Inject;
import java.io.Serializable;

public class CandidatoService extends ServiceAbstract<Candidato> implements Serializable {

	private static final long serialVersionUID = 6892125671661361672L;
	private CandidatoDao dao;

	@Inject
	public CandidatoService(CandidatoDao dao) {
		super(dao);
		this.dao = dao;
	}

}

File: src/main/java/service/CandidatoService.kt
package service

import dao.CandidatoDao
import model.Candidato
import java.io.Serializable
import javax.inject.Inject

class CandidatoService @Inject constructor(var daoU: CandidatoDao) : Serializable, AbstractService<Candidato>(dao = daoU) {

    val serialVersionUID = -2800411759544395063L

    fun listarTodosOsCandidatos(): List<Candidato> {
        return daoU.listarTodosComAvaliacoes()
    }

}


